ui/app/rif/components/tokens/index.js

```diff
@@ -0,0 +1,58 @@
+import React from 'react';
+import {DEFAULT_ICON, PATH_TO_RIF_IMAGES, JOINED_TEXT, UNJOINED_TEXT} from '../../constants';
+import {FontAwesomeIcon} from '@fortawesome/react-fontawesome';
+import {brandConnections} from '../../constants/icons';
+import ChannelChiplet from './ChannelChiplet';
+
+const Logo = (className, icon) => {
+  return (<div className={(className && className.tokenLogo) ? className.tokenLogo : ''}>
+    {icon && icon.icon ?
+      <img src={PATH_TO_RIF_IMAGES + icon.icon}
+           className={(className && className.tokenLogoPng) ? className.tokenLogoPng : ''}/>
+      :
+      <FontAwesomeIcon icon={DEFAULT_ICON.icon} color={DEFAULT_ICON.color}
+                       className={(className && className.tokenLogoIcon) ? className.tokenLogoIcon : ''}/>
+    }
+  </div>);
+}
+
+const Channels = (className, channelsLength) => {
+  return (<div>
+    <div className={(className && className.tokenInfoChannels) ? className.tokenInfoChannels : ''}>
+      <div className={(className && className.tokenInfoChannelsIcon) ? className.tokenInfoChannelsIcon : ''}>
+        <FontAwesomeIcon icon={brandConnections.icon} color={brandConnections.color} />
+      </div>
+      <div className={(className && className.tokenInfoChannelsQty) ? className.tokenInfoChannelsQty : ''}>{channelsLength}</div>
+    </div>
+  </div>);
+}
+
+const JoinedChip = (className, joined) => {
+  return (<div className={
+    ((className && className.tokenInfoStatus) ? className.tokenInfoStatus : '') +
```

Comment:
What is the idea using the plus operator here?

Reply:
This is for setting the styles. It's just for the different type of styles that this component has. I know it looks weird, but it's just to re-use the JoinedChip, and apply styles depending on the joined att